addons/mail/static/src/components/discuss_sidebar_category_item/discuss_sidebar_category_item.scss

```diff
@@ -0,0 +1,124 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_DiscussSidebarCategoryItem {
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    padding: map-get($spacers, 2) 0;
+
+    &:hover .o_DiscussSidebarCategoryItem_commands {
+        display: flex;
+    }
+}
+
+.o_DiscussSidebarCategoryItem_command {
+    margin-left: $o-mail-discuss-sidebar-category-item-margin;
+    margin-right: $o-mail-discuss-sidebar-category-item-margin;
+
+    &:first-child {
+        margin-left: 0px;
+    }
+
+    &:last-child {
+        margin-right: 0px;
+    }
+}
+
+.o_DiscussSidebarCategoryItem_commands {
+    display: none;
+}
+
+.o_DiscussSidebarCategoryItem_image {
+    width: map-get($sizes, 100);
+    height: map-get($sizes, 100);
+}
```

Comment:
this is missing object-fit: cover; so when image is not a square it has bad ratio